### ISSUE TITLE
Multi-GPU Training Issue

### ISSUE THREAD

**Issue by richard3333p** [AlexeyAB/darknet#8714]
I'm trying custom training in multi-GPU environment.
I have a question regrading the amount of used images for training at the same time.

My training environment is below.
- DGX V100(Tesla V100(32G) x4)
- Ubuntu 20.04
- YOLOv4

Common condition is to set the batch size= 64, subdivision=8
I has increased the number of GPUs(#0~3) and then the number of images was increased 64, 128, 192, 256 per each iteration.

However, the number of processed images was diffierent at the same time.

1. When I set "-gpus 0", one iteration was prosecced.
-The number of used images increased in increments of 64 : 64 images -> 128 images -> 192 images -> 256 images 

2. When I set "-gpus 0,1", two iterations are prosecced at the same time. 
-The number of used images increased in increments of 256 : 256 images -> 512 images -> 768 images -> 1024 images

3. When I set "-gpus 0,1,2", three iterations are prosecced at the same time. 
-The number of used images increased in increments of 576 : 576 images -> 1152 images -> 1728 images -> 2304 images

4. When I set "-gpus 0,1,2,3", four iterations are prosecced at the same time. 
-The number used image increased in increments of 1024 : 1024 images -> 2048 images -> 3072 images -> 4096 images

Were there some gradient accumulation rule in multi-gpu environmnet?

Thanks in advance.
